src/extension/background-script/db.ts

```diff
@@ -19,6 +19,10 @@ class DB extends Dexie {
     this.version(2).stores({
       blocklist: "++id,host,name,imageURL,isBlocked,createdAt",
     });
+    this.version(3).stores({
+      payments:
+        "++id,allowanceId,host,location,name,description,totalAmount,totalFees,preimage,paymentRequest,metadata,paymentHash,destination,createdAt",
```

Comment:
still thinking about leaving `payments` as it is and adding a new table metadata with the metadata entries associated to a payment.

this would leave the current data in payment untouched and we can easily always extend the metadata table with new columns independent of the rest of the app

Reply:
![image](https://user-images.githubusercontent.com/55848322/184352456-9788e56e-fcc4-4aba-8016-5bcc8da0691d.png)